fvm/blueprints/scripts/systemChunkTransactionTemplate.cdc

```diff
@@ -17,5 +18,10 @@ transaction {
             .borrow<&RandomBeaconHistory.Heartbeat>(from: RandomBeaconHistory.HeartbeatStoragePath)
             ?? panic("Couldn't borrow RandomBeaconHistory.Heartbeat Resource")
         randomBeaconHistoryHeartbeat.heartbeat(randomSourceHistory: randomSourceHistory())
+
+        let evmHeartbeat = serviceAccount.storage
```

Comment:
Just a note here for the HCU:

- EVM contract should be updated first
- an EVMHeartbeat resource should be created (it wont automatically exist, because the updated init function will not be called, because the contract is already deployed)
- only then should the code be updated

Reply:
what do you think if I change this method to not panic for now if resource doesn't exist ?

Reply:
or can we update the contract and create the resource before the HCU ?

Reply:
actually we have 2 different cases... 
HCU on previewnet
Spork on other networks

for the HCU we could deploy the contract before the HCU, but we cannot do the same process during the spork.

So I think we do have to change the system contract to not panic on missing resource.

Reply:
updated to skip if resource is missing